test/integration/serverless/run.sh

```diff
@@ -138,6 +161,7 @@ trace_functions=(
     "trace-python"
     "trace-java"
     "trace-go"
+    # traces are not supported for .net lambda function on ARM64 architecture (snapshot is empty)
```

Comment:
We actually don't support .NET at all on ARM -- would it be possible to skip .NET tests for the ARM architecture?